.github/workflows/biotestmine.yml

```
@@ -0,0 +1,63 @@
+---
+# yamllint disable rule:line-length
+name: Biotestmine
+# yamllint disable-line rule:truthy
+on:
+    push:
+        paths:
+            - '.env'
+            - '.github/workflows/biotestmine.yml'
+            - 'intermine_builder/**'
+            - 'local.docker-compose.yml'
+            - 'mkdatadirs.sh'
+            - 'postgres/**'
+            - 'solr/**'
+            - 'tomcat/**'
+
+jobs:
+    biotestmine:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Install prerequisites
+              run: |
+                  set -euo pipefail
+                  sudo apt -y install wait-for-it
+            - name: Make data dirs
+              run: |
+                  set -euo pipefail
+                  cd ${GITHUB_WORKSPACE}
+                  ./mkdatadirs.sh
+                  mkdir ./data/mine/biotestmine
```

```suggestion
                  mkdir ./data/mine/biotestmine
```
is this required? `mkdatadirs.sh` script already creates this directory.

---

I see, you have changed the mkdatadirs script as well. Ignore this suggestion.